### PR TITLE
Fix/commonjs export

### PR DESCRIPTION
**Pull request by michielmetcake**
Fixes:
https://github.com/imagemin/imagemin-svgo/issues/56
and 
https://github.com/imagemin/imagemin-svgo/issues/51

### CODE REVIEW

**Review by PizzaPete — APPROVED** [review]

**Comment by webketje** [comment]
@michielmetcake Unfortunately this PR breaks all tests and is the opposite direction of where sindre intended his packages to go, see the FAQ section of https://github.com/sindresorhus/meta/discussions/15

> Do you plan to include CommonJS fallback files?
> No. Node.js 12 supports ESM natively. There's no point.

So reverting to the .cjs format is not an option. If you use imagemin directly, you can do as below for a workaround:

```js
const loadImageminSvgo = await import('imagemin-svgo')
const imageminSvgo = loadImageminSvgo.default
```
If you depend on a package which expects imagemin-svgo to export a CJS module, you can only insist that the maintainers of the depending package implement this fix or migrate to another CJS-supported implementation.

**Comment by sindresorhus** [comment]
https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c

**Comment by PizzaPete** [comment]
We're no longer blocked by this. That's why there was no activity in this PR for a while.

**Pull request closed without merging**
